wagtailmenus/forms.py

```diff
@@ -11,7 +11,9 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         if app_settings.FLAT_MENUS_HANDLE_CHOICES:
             self.fields['handle'] = forms.ChoiceField(
```

Comment:
While we're at it, I wonder if a better approach here would be to just override the widget, rather than the entire field? Does the following solve the problem for you?

```
self.fields['handle'].widget = forms.Select(choices=app_settings.FLAT_MENUS_HANDLE_CHOICES)
```